tracing/tracing-test.sh

```diff
@@ -23,17 +32,48 @@ jaeger_server=${jaeger_server:-localhost}
 jaeger_ui_port=${jaeger_ui_port:-16686}
 jaeger_docker_container_name="jaeger"
 
+logdir=""
+
 # Cleanup will remove Jaeger container and
 # disable tracing.
-cleanup(){
-	stop_jaeger 2>/dev/null || true
-	"${SCRIPT_PATH}/../.ci/configure_tracing_for_kata.sh" disable
-}
+cleanup()
+{
+	local fp="info"
+	local result="failed"
+	local dest="$logdir"
+
+	if [ "$success" = true ]; then
+		result="passed"
+
+		[ -z "$DEBUG_KEEP_JAEGER" ] && stop_jaeger 2>/dev/null || true
 
-trap cleanup EXIT
+		# The tests worked so remove the logs
+		if [ -n "$DEBUG" ]; then
+			eval "$fp" "test $result - logs left in '$dest'"
+		else
+			"${SCRIPT_PATH}/../.ci/configure_tracing_for_kata.sh" disable
```

Comment:
Just to check I understood the code organization. Even though the script (e.g.  `configure_tracing_for_kata.sh`) is called (made) by a single test suite, the convention is that it lives under `tests/.ci` because any test system configuration script should there. Did I get it correctly?